lib/ransack/helpers/form_helper.rb

```diff
@@ -110,11 +110,11 @@ def initialize(search, attribute, args, params)
         end
 
         def up_arrow
-          '&#9660;'.freeze
+          Ransack.options[:up_arrow]
         end
 
         def down_arrow
-          '&#9650;'.freeze
+          Ransack.options[:down_arrow]
         end
```

Comment:
I originally extracted these 2 public arrow methods out from the private method #order_indicator to allow developers to set custom arrows by overloading them.

Now that developers can do it with a config setting, I'm not sure there is any reason to keep them as separate public methods. Performance-wise it would be better to inline them.

WDYT?